commit 592f5e5c2b3ca5d3ad4862cef9dc4f9af4760781
Author: Daniel Alley <dalley@redhat.com>
Date:   Tue Dec 19 15:45:21 2023 -0500

    temp2

diff --git a/tests/assets/SOURCES/example_data.xml b/tests/assets/SOURCES/example_data.xml
index 7d0faf0..4add8dd 100644
--- a/tests/assets/SOURCES/example_data.xml
+++ b/tests/assets/SOURCES/example_data.xml
@@ -2,4 +2,7 @@
 <content>
 Hey  there, this is a lil somethin somethin for the test rpm.
 </content>
-</xml>
\ No newline at end of file
+<content>
+And now I'm adding more
+</content>
+</xml>
diff --git a/tests/assets/SOURCES/multiplication_tables.py b/tests/assets/SOURCES/multiplication_tables.py
index f01af9a..1493675 100644
--- a/tests/assets/SOURCES/multiplication_tables.py
+++ b/tests/assets/SOURCES/multiplication_tables.py
@@ -2,4 +2,4 @@

 for x in range(1, 11):
     for y in range(1, 11):
-        print('%d * %d = %d' % (x, y, x*y))
+        print("{} * {} = {}".format(x, y, x*y))
index e69de29..0000000
